cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/detector/TopicAnomalyFinder.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2020 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.detector;
+
+import com.linkedin.cruisecontrol.common.CruiseControlConfigurable;
+import java.util.Set;
+import org.apache.yetus.audience.InterfaceStability;
```

Comment:
Is this import intentional? Shouldn't we use `org.apache.kafka.common.annotation.InterfaceStability;`?